HomePage.html

```diff
@@ -61,13 +61,226 @@ <h1>Hey there. I'm Mariane <br> <span>I'm a software developer </span> </h1>
 						</ul>
 
 					<button class="scroll-button">
-						<img src="Assets/Icons/scroll-button.svg" alt=""/>
+						<a href="#projects"><img src="Assets/Icons/scroll-button.svg" alt=""/></a>
 					</button>
```

Comment:
[REQUIRED] Kindly ensure that your button states match the template.

|Template|
|-----------|
|![image](https://user-images.githubusercontent.com/62938735/192018711-8fce6d47-a744-49e0-8755-b2634fb467b5.png)|

Reply:
@fobadara I think this is okay because its the scroll down icon and not an actual button that will change states??

Reply:
@fobadara i've changed it to a div just in case.